maintainers/maintainer-list.nix

```
@@ -14508,6 +14508,12 @@
     githubId = 38578268;
     name = "Morgan Wolfe";
   };
+  mx2uller = {
+    email = "mx2uller@proton.me";
+    github = "mx2uller";
+    githubId = 93703653;
+    name = "Marko Muller";
```

```suggestion
    name = "Marko Müller";
```
I think we can handle unicode symbols here :)